src/containers/my-resources/edit-attachment-modal/EditAttachmentModal.constants.ts

```diff
@@ -1,25 +1,27 @@
-import { Attachment } from '~/types'
+import { Attachment, EditAttachmentForm } from '~/types'
 import { emptyField, textField } from '~/utils/validations/common'
 
-export const getInitialValues = (attachment: Attachment) => {
-  const initialFileName = attachment.fileName.substring(
-    0,
-    attachment.fileName.lastIndexOf('.')
-  )
+export const getInitialValues = (
+  attachment: Attachment
+): EditAttachmentForm => {
+  const [initialFileName, fileExtension] = attachment.fileName.split('.')
```

Comment:
I think it is not correct solution. For example attachment's filename can be `my.attachment-present-simple.pdf`. If it is not checked before this function is used, variables will have following values:
- `initialFileName`: `my`
- `fileExtension`: `attachment-present-simple.pdf`

Please make sure it is checked before this splitting is used or write better solution to split file to filename and extension

Reply:
according to current fixes, now we have an object returned from this function with `fileName` and `fileExtension`

Reply:
Nice work!